readthedocs-theme/templates/readthedocs/company.html

```diff
@@ -0,0 +1,276 @@
+                <a class="ui avatar" href=""><img class="ui avatar image" src="/theme/img/team/default-avatar.svg"></a>
+                <a class="ui avatar" href=""><img class="ui avatar image" src="/theme/img/team/default-avatar.svg"></a>
+                <a class="ui avatar" href=""><img class="ui avatar image" src="/theme/img/team/default-avatar.svg"></a>
+                <a class="ui avatar" href=""><img class="ui avatar image" src="/theme/img/team/default-avatar.svg"></a>
+                <a class="ui avatar" href=""><img class="ui avatar image" src="/theme/img/team/default-avatar.svg"></a>
+                <a class="ui avatar" href=""><img class="ui avatar image" src="/theme/img/team/default-avatar.svg"></a>
+                <a class="ui avatar" href=""><img class="ui avatar image" src="/theme/img/team/default-avatar.svg"></a>
+                <a class="ui avatar" href=""><img class="ui avatar image" src="/theme/img/team/default-avatar.svg"></a>
+                <a class="ui avatar" href=""><img class="ui avatar image" src="/theme/img/team/default-avatar.svg"></a>
+                <a class="ui avatar" href=""><img class="ui avatar image" src="/theme/img/team/default-avatar.svg"></a>
+                <a class="ui avatar" href=""><img class="ui avatar image" src="/theme/img/team/default-avatar.svg"></a>
+                <a class="ui avatar" href=""><img class="ui avatar image" src="/theme/img/team/default-avatar.svg"></a>
+                <a class="ui avatar" href=""><img class="ui avatar image" src="/theme/img/team/default-avatar.svg"></a>
+              </div>
+              <div class="eleven wide column">
+                <p>
+                  We are honored to have several external contributors that have kept with us for a long time and helped us a lot along the way. Here we list just a few but the full list can be found at <a href="https://github.com/readthedocs/readthedocs.org/graphs/contributors">Github</a>.
+                </p>
+                <p>
+                  A heart felt thanks to all of you!
+                </p>
+              </div>
+            </div>
+          </div>
+
+        </div>
+      </div>
+
+    </div>
+  </section>
+
+
+  <!-- Resources -->
+  <section class="ui vertical light segment padded" id="resources">
+
+    <div class="ui container">
+      <div class="ui grid stackable three column">
+
+        <div class="column">
+          <div class="ui segment basic">
+            <h3 class="ui header huge">Our Blog</h3>
+            <p>
+              This is where we post all the week to week progress updates and announcements.
+            </p>
+          </div>
+          <div class="ui segment basic mini">
+            <h6 class="ui header">Latest blog posts:</h6>
+            <div>
+              <b><a href="https://blog.readthedocs.com/newsletter-december-2021/">Read the Docs newsletter - December 2021</a></b> 
```

Comment:
Are we automating this listing here to? It's also fine to just link to the blog section, or return to this when we can automate a blog listing.

Reply:
Done here https://github.com/readthedocs/site-community/pull/35/commits/e0c5dd497d5bc240d866045684923d8d3532aabb. I used the text "Check it out" for the blog link.

Reply:
The middle column, talking about the docs, has now a very disproportional text length, but I'm not sure if we should revise that copy accordingly only to change it back as soon as the blog is done and this content is updated back.